examples/rosettacode/pig_the_dice_game.nit

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: A+B
+# SEE: http://rosettacode.org/wiki/Pig_the_dice_game
+
+var player = 0
+var score = [0, 0]
+var turnscore = 0
+var max_score = 100
+loop
+	if (score[player] + turnscore) >= max_score then
+			print "Congratulation Player {player + 1}! You won with {score[player]} points! "
+			return
+	end
+	print "Player {player +1}"
+	print "Total score : {score[player]}"
+	print "What do you want to do? (\"1\" to roll, \"0\" to hold)"
+	var input = gets
+	assert not stdin.eof
```

Comment:
~~~nit
if stdin.eof then break
~~~

If I Ctrl-D I want to exit, not to crash